src/styles/main.scss

```diff
@@ -1,3 +1,83 @@
+@import "./variables.scss";
+
 body {
   margin: 0;
 }
+
+* {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  gap: 1px;
+  margin-top: 100px;
+  margin-left: $general-margin;
+  margin-right: $general-margin;
+  text-align: center;
+  
+  &__elem {
+    display: block;
+    width: 98px;
+    height: 98px;
+    background-color: $color-main;
+    border: 1px solid $color-border;
+    transition: 500ms;
```

Comment:
Better to create a variable for this value. Also, please specify to which exactly property you want to apply transition because now it applies to all the properties and it's really bad for performance
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x